src/components/Controls.vue

```diff
@@ -338,9 +338,9 @@ export default {
             </v-row>
           </v-alert>
 
-          <node-link
+          <multi-link
             v-if="provenance"
-            ref="nodelink"
+            ref="MultiLink"
```

Comment:
As essentially a variable in your program, this should follow the JavaScript/TypeScript conventions and be all lowercase (exactly as you had it in the MultiMatrix renaming operation).

```suggestion
            ref="multilink"
```

You will need to update the capitalization wherever `this.$refs.MultiLink` appears elsewhere in this PR.